src/store/modules/shipment/mutations.ts

```diff
@@ -4,10 +4,15 @@ import * as types from './mutation-types'
 
 const mutations: MutationTree <ShipmentState> = {
   [types.SHIPMENT_LIST_UPDATED] (state, payload) {
-      state.shipments.list = payload.shipments;
```

Comment:
Sir, if we will revert this line of code, this will overwrite our shipment list each time we will click on load more shipment.
that's why I have assigned these shipments to their shipment id as key.